### ISSUE TITLE
Empty results on table creating duplicate div

### ISSUE THREAD

**Issue by dougbarrett** [vuematerial/vue-material#2017]
### Steps to reproduce

You can reproduce on the live demo link provided in the docs:

https://codesandbox.io/s/p7n7xjjr3j?module=App.vue

### Which browser?

Chome 71
MacOS 10.14.3
    vue 2.5.22
    vue-material 1.0.0-beta-10.2
    vue-router 3.0.2

### What is expected?

It should display the contents of the `md-table-empty-state` component right below the table heading.

### What is actually happening?

It's creating a div with the following classes:

```.md-content.md-table-content.md-scrollbar.md-theme-default.md-table-empty```

with a static height that is pushing down the custom component created.

It appears this only happens when you set the `md-fixed-header` attribute on the `md-table` component.

### Reproduction Link

https://codesandbox.io/s/p7n7xjjr3j?module=App.vue

**Comment by Samuell1** [comment]
Yeah this is maybe because Vue got updante and in there are changes for slots looks like.
